server/data/tokenVerification.test.ts

```diff
@@ -7,6 +7,7 @@ describe('token verification api tests', () => {
   let fakeApi: nock.Scope
 
   beforeEach(() => {
+    config.apis.tokenVerification.url = 'http://localhost:8100'
```

Comment:
fixing this test as it fails when a local .env file is present - should probably move to just injecting the url